content-security-policy/script-src/script-src-strict_dynamic_hashes.html

```diff
@@ -47,6 +47,18 @@ <h1>`strict-dynamic` allows scripts matching hashes present in the policy.</h1>
             document.body.appendChild(e);
         }, 'Script injected via `appendChild` from a script matching SHA256 hash is allowed with `strict-dynamic`.');
     </script>
+
+    <script nonce='dummy'>
+        var externalRan = false;
+    </script>
+    <script src='./externalScript.js'
+        integrity="sha256-wIc3KtqOuTFEu6t17sIBuOswgkV406VJvhSk79Gw6U0="></script>
+    <script nonce='dummy'>
+        async_test(function(t) {
```

Comment:
I think this can just be `test`, since nothing async is happening:

```
test(() => {
    assert_true(externalRan);
}, "External script...");
```

Reply:
Ah right, makes sense to keep it simple. Changed it now in ed50066712e82bca4026953910fbf5d5f42e8eaa